src/components/schedule/Schedule.tsx

```diff
@@ -80,44 +76,60 @@ const Schedule = ({
     periodID: number;
   }) => void;
   toggleFetched?: () => void;
-}): JSX.Element => (
-  <div
-    className="scroll-w-0 flex flex-row gap-5 overflow-x-auto overflow-y-hidden
-      !px-0 sm:overflow-x-visible"
-  >
-    <div aria-hidden className="flex flex-col gap-2 pt-4 pb-2 pl-4 sm:pl-0">
-      {schedule.content.map((scheduleRow) => (
-        <ScheduleDay key={scheduleRow.day} day={scheduleRow.day} />
-      ))}
-    </div>
+}> = ({
+  schedule,
+  role,
+  noScroll,
+  allowEdit,
+  setAddPeriod,
+  setEditPeriod,
+  setDeletePeriod,
+  toggleFetched,
+}) => {
+  const periodWidth = 112;
+
+  return (
     <div
-      className={
-        noScroll
-          ? "grow"
-          : "scroll-w-0 scroll-desktop grow overflow-y-clip sm:overflow-x-auto"
-      }
+      className="scroll-desktop flex w-full flex-row gap-5 overflow-x-auto
+      overflow-y-hidden !px-0 sm:overflow-x-visible"
     >
-      <LayoutGroup>
-        <AnimatePresence initial={false}>
-          <ul className="flex flex-col gap-2 pt-4 pb-2 pl-1 pr-4 sm:pr-0">
-            {schedule.content.map((scheduleRow) => (
-              <ScheduleRow
-                key={scheduleRow.day}
-                scheduleRow={scheduleRow}
-                role={role}
-                allowEdit={allowEdit}
-                setAddPeriod={setAddPeriod}
-                setEditPeriod={setEditPeriod}
-                setDeletePeriod={setDeletePeriod}
-                toggleFetched={toggleFetched}
-              />
-            ))}
-          </ul>
-        </AnimatePresence>
-      </LayoutGroup>
+      <div aria-hidden className="flex flex-col gap-2 pb-2 pl-4 sm:pl-0">
+        <div className="mb-1 h-[2.375rem]" />
+        {schedule.content.map((scheduleRow) => (
+          <ScheduleDay key={scheduleRow.day} day={scheduleRow.day} />
```

Comment:
If you check out the preview, you’ll see that the days aren’t aligned with the contents. The height of a Schedule Day is not the same as that of a Schedule Period because it doesn’t account for the height gained from the borders (total add 4px).

Reply:
It doesn't seem to happen on my end, which device are you using to view this, for me, it shows the height as`60px` for both the content and the day.

Reply:
Safari on iPadOS 16.2.